pkg/libovsdb/schema.go

```diff
@@ -71,3 +79,344 @@ func (schema DatabaseSchema) validateOperations(operations ...Operation) bool {
 	}
 	return true
 }
+
+// TableSchema is a table schema according to RFC7047
+type TableSchema struct {
+	Columns map[string]*ColumnSchema `json:"columns"`
```

Comment:
why do you use pointer to ColumnSchema ?

Reply:
original code (I generally agree with your comment)